modules/nf-core/chopper/main.nf

```diff
@@ -0,0 +1,43 @@
+process CHOPPER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::chopper=0.3.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/chopper:0.3.0--hd03093a_0':
+        'quay.io/biocontainers/chopper:0.3.0--hd03093a_0' }"
+
+    input:
+    tuple val(meta), path(fastq)
+
+    output:
+    tuple val(meta), path("${prefix}.fastq.gz") , emit: fastq
+    path "versions.yml"                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args  = task.ext.args  ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def args3 = task.ext.args3 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    if ("$fastq" == "${prefix}.fastq.gz") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+
+    """
+    zcat
+        $args \\
+        $fastq | \\
+    chopper \\
+        --threads $task.cpus \\
+        $args2 | \\
+    gzip \\
+        $args3 >  ${prefix}.fastq.gz
```

Comment:
```suggestion
        $args3 > ${prefix}.fastq.gz
```

Reply:
should be fixed now, squashed the commit into the previous one and force pushed onto the chopper branch to have a nicer history